subworkflows/local/abacas_multifasta.nf

```diff
@@ -0,0 +1,63 @@
+//
+// Process multiple multifasta files with abacas
+//
+include { ABACAS        } from '../../modules/nf-core/abacas/main'
+include { SEQKIT_SPLIT  } from '../../modules/local/seqkit_split'
+include { SEQKIT_CONCAT } from '../../modules/nf-core/seqkit/concat/main'
+
+workflow ABACAS_MULTI {
+    take:
+    scaffold   // channel: [ val(meta), path(scaffold) ]
+    multifasta // channel: /path/to/genome.fasta
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    //
+    // Split multifasta file into individual fasta files
+    //
+    SEQKIT_SPLIT (
+        multifasta.map { [ [:], it ] }
+    )
+    ch_fasta_list = SEQKIT_SPLIT.out.fastx
+    ch_versions = ch_versions.mix(SEQKIT_SPLIT.out.versions)
```

Comment:
We should consider using `.splitFasta()` instead, saves us an extra module to run. 
https://www.nextflow.io/docs/latest/reference/operator.html#splitfasta